explore/src/main/scala/explore/tabs/SiderealTargetEditorTile.scala

```diff
@@ -25,7 +25,7 @@ import java.time.Instant
 
 object SiderealTargetEditorTile {
 
-  def siderealTargetEditorTile(
```

Comment:
Why the renaming? `sidereal` is correct, isn't it?

Reply:
Good catch. That's a mistake, I must have renamed it something else and then renamed it back.